editor/store.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * External dependencies
+ */
+import { applyMiddleware, createStore } from 'redux';
+import refx from 'refx';
+import multi from 'redux-multi';
+import { flowRight } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import effects from './effects';
+import reducer from './reducer';
+import { getPreferences } from './selectors';
+
+/**
+ * Module constants
+ */
+const GUTENBERG_PREFERENCES_KEY = 'GUTENBERG_PREFERENCES';
+
+/**
+ * Loads the initial preferences and saves them to the local storage once changed
+ * @param {Object} store Redux Store
+ */
+function setupStorePersistence( store ) {
```

Comment:
Again, maybe getting ahead of myself, but thinking about how this might be implemented for maintainability. [Redux Persist](https://github.com/rt2zz/redux-persist) seems to be the canonical reference here, though I think it's overkill for what we need. That said, maybe their patterns of reducer key whitelisting and rehydration actions could serve as good inspiration?

Also, this seems to fall under the pattern of what I'd consider to be a store enhancer: http://redux.js.org/docs/api/createStore.html . It could fit well within what we're already creating as an `enhancers` array below (a store enhancer accepts `createStore` and returns a new `createStore`, [example](https://github.com/reactjs/redux/blob/f267dd02d19c2ede88e37b8a73cc0b0b63413be2/src/applyMiddleware.js#L20-L36)).